storybook/stories/playground/box/index.js

```diff
@@ -9,10 +9,28 @@ import {
 	BlockToolbar,
 } from '@wordpress/block-editor';
 
+/**
+ * External dependencies
+ */
+import componentsStyles from '!!raw-loader!@wordpress/components/build-style/style.css?raw';
+import blockEditorContentStyles from '!!raw-loader!@wordpress/block-editor/build-style/content.css?raw';
+import blocksStyles from '!!raw-loader!@wordpress/block-library/build-style/style.css?raw';
+import blocksEditorStyles from '!!raw-loader!@wordpress/block-library/build-style/editor.css?raw';
+
 /**
  * Internal dependencies
  */
 import editorStyles from '../editor-styles';
+
+//Base styles for the content within the block canvas iframe.
+const contentStyles = [
+	{ css: componentsStyles },
+	{ css: blockEditorContentStyles },
+	{ css: blocksStyles },
+	{ css: blocksEditorStyles },
+	{ css: editorStyles },
```

Comment:
`editorStyles` is [a styles array](https://github.com/WordPress/gutenberg/blob/16146bad6f2aa776232d69f3c6030c8eab329377/storybook/stories/playground/editor-styles.js) so unless we change its export to a simple string, it would need to be used like so:
```suggestion
	...editorStyles,
```

Reply:
Ah, correct, Thanks @stokesman I will update this too and add comment.

Reply:
Fixed